app/src/main/java/com/globant/example/mentorapp/subscriberDetails/presentation/view/adapter/UserRepositoriesAdapter.java

```diff
@@ -0,0 +1,59 @@
+package com.globant.example.mentorapp.subscriberDetails.presentation.view.adapter;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.globant.example.mentorapp.R;
+import com.globant.example.mentorapp.subscriberDetails.domain.model.RepositoryModel;
+
+import java.util.List;
+
+/**
+ * Custom adapter to show user repositories by name and url.
+ * Created by oscar.amezquita on 12/07/2017.
+ */
+
+public class UserRepositoriesAdapter extends RecyclerView.Adapter<UserRepositoriesAdapter.ViewHolder> {
+
+    private List<RepositoryModel> userRepositoriesAdapterList;
+    private Context context;
```

Comment:
Is used to get a zero constant  `context.getResources().getInteger(R.integer.zero_constant)`